submissions/asaMitaka/FriendsApp/script.js

```diff
@@ -0,0 +1,177 @@
+const url = 'https://randomuser.me/api/?results=12';
+
+let aToZLast = document.querySelector('.sortedAtoZLast');
+let zToALast = document.querySelector('.sortedZtoALast');
```

Comment:
There are a lot of `querySelector`s here. Instead, you can just get your `form` element and then access any field of the form at any time by its `name`. 

So for example, you can have a couple of radiobuttons that share the same `name`, let's say "gender". Each of them should have a `value` attribute, like `male` and `female` for example.

So then you can just go
```js
const form = document.querySelector("#myForm");
console.log(form.gender) // this should return male\female based on checked radiobutton
```